share/docker/Dockerfile

```diff
@@ -109,10 +105,12 @@ RUN curl -L https://cpanmin.us | perl - --self-upgrade && \
 RUN wget -O /usr/local/bin/phpunit https://phar.phpunit.de/phpunit-5.7.phar && chmod +x /usr/local/bin/phpunit
 
 # Install Python2 packages
-RUN pip install zstandard
+RUN python2 -m pip install --upgrade pip setuptools \
+ && python2 -m pip install zstandard
```

Comment:
@sekikn I think we should put any of these compressors into extras_require in the setup files so that users can do like `pip3 install avro-python3[zstandard]` just like they can do `pip install avro[snappy]` now. I'll open a future PR for that. What do you think?

Reply:
Sounds good to me!